### DISCUSSION TITLE
Mudraptor beak limb does loopdy-loops (no weldjoint)

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14618] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Mudraptor beak isnt using weldjoint and just does loops and swings

https://github.com/user-attachments/assets/7f8c9d4d-4016-4b0e-9cb1-82bcdf5eb494

Rotate beak by 90° 
![261718221-f4ff77c3-0bfc-492c-b90b-274619b27f2a](https://github.com/user-attachments/assets/efdeddbe-05c1-4bc8-979e-8dd92a8c4335)

```xml
    <sprite sourcerect="410,18,100,67" origin="0.5,0.5" depth="0.049" color="230,200,150,255" deadcolor="230,200,150,255" deadcolortime="0" texture="Content/Items/Genetic/Genetic.png" ignoretint="true" />

  <joint name="Head to Mudraptor Beak0" limb1="1" limb2="13" limb1anchor="0,0" limb2anchor="-20,-6" lowerlimit="0" upperlimit="0" weldjoint="True"  canbesevered="False" limitenabled="True" />
```

### Reproduction steps

attack with mudraptor beak (try aiming to the bottom right)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested, mudraptor beak is very rigid now. Closing discussion.
